runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -1232,32 +1232,38 @@ def DeleteCommands()
   delcommand Var
   delcommand Winbar
 
-  if exists('k_map_saved')
-    if !empty(k_map_saved) && !k_map_saved.buffer
+  if k_map_saved isnot null_dict
+    if !empty(k_map_saved) && k_map_saved.buffer
+      # pass
+    elseif !empty(k_map_saved) && !k_map_saved.buffer
       nunmap K
       mapset(k_map_saved)
     elseif empty(k_map_saved)
-      nunmap K
+      silent! nunmap K
```

Comment:
why the silent!

Reply:
there maybe buf local mapping at source, so e.g gdb had no 'K' mapping actually.

Reply:
there maybe buf local mapping at source window, then e.g gdb window had no 'K' mapping actually.

Reply:
alright then, thanks. Can you please make the suggested changes from @errael ?

Reply:
that's a confusion to himself, https://github.com/vim/vim/pull/15013#discussion_r1641334643 had explained.
// though you accepted his PR vs/over mine in the past about this null/null_obj topic, but just a code style.
// vs this termdebug you had accepted null_obj for `saved_mousemodel`, a similar flag, so i just kept the same.

Reply:
that's a confusion to himself, https://github.com/vim/vim/pull/15013#discussion_r1641334643 had explained.
// though you accepted his PR vs/over mine in the past about this null/null_obj topic, but just a code style.
// vs this termdebug you had accepted null_obj for `saved_mousemodel`, a similar flag, so i just kept the same.

Reply:
ah, okay. So you initialize to null_dict and then testing against null_dict should be correct.